Test case (#2)
Azure-Samples/azure-samples-js-management: samples/compute/README.md

---
page_type: sample
languages:

> - typescript

Reviewer: ```suggestion
- typescript
- javascript
```

Reviewer: @colawwj  Please change other README.md files accordingly. Thanks

products:
- azure

> description: "These code samples will show you how to manage Compute using Azure SDK for Typescript."

Reviewer 2: Azure SDK for JavaScript/TypeScript

> urlFragment: compute

Reviewer 2: Can we double check this? 
languages:
javascript or typescript?

Reviewer: Actually, it can support multple language configuration, but these samples themself are written by typescipt. So I was't sure whether we need to add javascript here.

Reviewer 2: could you double check data-plane Track 2 JS samples?

Reviewer 2: as long as it's consistent

---

# Getting started - Managing Compute using Azure typescript SDK

> These code samples will show you how to manage Compute using Azure SDK for Typescript.

Reviewer 2: Azure SDK for JavaScript/Typescript for all files

## Features

This project framework provides examples for the following services:

### Compute
* Using the Azure SDK for Typescript - Compute Management Library [@azure/arm-compute](https://www.npmjs.com/package/@azure/arm-compute) for the [Azure Compute API](https://docs.microsoft.com/en-us/rest/api/compute/)

### Installation

> 1.  Before using the sdk packgae,we need [install nodejs](https://nodejs.org/en/download/) and add it into environment variables.

Reviewer 2: typo.

maybe do a typo check on all files

Reviewer: ```suggestion
1.  Before using the sdk package, we need to [install nodejs](https://nodejs.org/en/download/) and add it into environment variables.
```

    ```
    nodejs version: 14.16.0
    ```
### Quickstart

1.  Clone the repository.

    ```
    git clone https://github.com/Azure-Samples/azure-samples-js-management
    ```

2.  Install the dependencies using npm.

    ```
    cd azure-samples-js-management/samples/compute

>     npm i typescript@4.1.3

Reviewer: ```suggestion
    npm i -g typescript@4.1.3 
```

>     npm i -g typescript@4.1.3

Reviewer 3: Do we need to specify the version here?

---
page_type: sample
languages:

> - JavaScript/Typescript

Reviewer: ```suggestion
- javascript
- typescript
```
